ibis/expr/datatypes/core.py

```diff
@@ -463,15 +462,15 @@ def is_variadic(self) -> bool:
 
 
 @public
-class Unknown(DataType, Singleton):
```

Comment:
`koerce.Annotable` now supports singleton, so this can be restored by using `class Unknown(DataType, singleton=True)`